_posts/2021-04-02-gs_00001_computer_setup.md

```diff
@@ -0,0 +1,439 @@
+is on so many machines and its users often do not have IT knowledge it is the
+target of many computer viruses. This is reinforced by its complexity which
+leads space for security issues. Therefore, do not click on dubious links or
+download things from untrusted websites (this applies to all systems!).
+
+Mac OS is the operating system of Apple. It is shipped with all Apple computers
+and intended to be used with Apple products only. There are ways to install it
+on other machines, but this can be very tricky. Mac is different from Windows,
+but still very easy and intuitive to use. In my opinion, it runs more smoothly
+than Windows and looks nicer. While Windows sometimes hangs up, I never
+experienced this on a Mac. A Linux-like terminal lets programmers work
+efficiently. The disadvantage is the price of Apple products and the limited
+choice of products.
+
+Linux is a family of open-source operating systems. There are many so-called
+Linux distributions available which all are build on top of the Linux kernel,
+which is something like the heart of Linux. The kernel is combined with
+different software like a graphical interface to build a distribution. So,
+internally all distributions work the same, but they appear differently to the
+user. If you talk about Linux, you usually talk about all kinds of
+distributions. Linux is very popular among software developers. It is free to
+use, the source code is available and you can customize it as much as you like.
+Since everyone can check the code, report failures and even contribute, it is a
+very stable and safe operating system. A terminal (we come to that in a later
+post) offers a way for programmers to efficiently make use of the whole system.
+Linux also offers a huge community, where you always can ask questions or chat
+with people about your projects. As a disadvantage, it might take longer to get
+used to it. This depends on the distribution you choose. Ubuntu is a well-known
+distribution that is also suitable for beginners. Arch Linux is probably the
+total opposite of it. In the basic version, you do not even have a graphical
+interface. In our blog, we use the [Manjaro](https://manjaro.org/) distribution.
+It builds on top of Arch Linux but is way easier to use. It tries to offer you
+the possibilities of Arch and at the same time keep the simplicity of Ubuntu.
+The advantage and disadvantage at the same time is that Manjaro will have the
+newest software releases. Advantage because you have new software features
+early. Disadvantage because you are likely to get undiscovered bugs. However,
+Manjaro maintains its own software repositories and will test software that is
+released by Arch before you will get it.
+
+Of course, you can go with any of the three operating systems. For the long
+term, I recommend going with Linux. Yes, it requires some time and reading to
+get used to it. At some point, however, it will make things easier, because many
```

Comment:
I feel like the point why Linux is especially useful in development (SW + HW), is missing here.

Reply:
Have the same feeling but hard to write it in 1 or 2 sentences. Let's talk about it